options/posix/generic/unistd-stubs.cpp

```diff
@@ -724,9 +728,49 @@ int getpagesize() {
 	return mlibc::page_size;
 }
 
-char *getpass(const char *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+// Code taken from musl
+// GLIBC extension for stdin/stdout
+char *getpass(const char *prompt) {
+	mlibc::infoLogger() << "\e[31mmlibc: getpass is broken, tcdrain is unavailable!\e[39m" << frg::endlog;
```

Comment:
It is probably a better idea to add a log to tcdrain() and let tcdrain() fail with ENOSYS. This function does not care about tcdrain()'s result anyway.